lib/manageiq/ui/classic/engine.rb

```diff
@@ -1,7 +1,16 @@
 require 'rails/all'
-require 'sass-rails'
-require 'coffee-rails'
-require 'patternfly-sass'
+
+if ENV["RAILS_ENV"] != "production" || defined?(Rake)
+  require 'sass-rails'
+  require 'coffee-rails'
+  require 'patternfly-sass'
+else
+  require 'bootstrap-sass/engine'
+  require 'font_awesome/sass/rails/engine'
```

Comment:
Assuming this doesn't break anything, I would not worry about it too much, we'll have to deal with these parts when moving to webpacker soon anyway..